docker/dev/Dockerfile

```diff
@@ -3,7 +3,7 @@ ARG RUST_BUILDER_IMAGE=ekidd/rust-musl-builder:1.47.0
 # Cargo chef plan
 FROM $RUST_BUILDER_IMAGE as planner
 WORKDIR /app
-RUN cargo install cargo-chef --version 0.1.6
+RUN cargo install cargo-chef
```

Comment:
Why are you removing the version name here? Isnt it better to get a deterministic version, instead of receiving random updates?

Reply:
IMO its prolly fine to not use a deterministic version here, the only reason we had to before was because they had a broken one that didn't support workspaces, and that's long been fixed now. If our builds fail in the future it'll become quickly apparent and we can go back to a deterministic version.

Reply:
What happens if Docker caches this command, and installs a different cargo-chef version in line 18 below? Isnt that gonna break things?

Reply:
I tested that out locally, it re-runs the planner and cacher if a new version is installed.

Reply:
I'm still a bit worried that there might be weird edge cases like this, but okay.